### DISCUSSION TITLE
iNav4.1 RC1 wiping ESCs

### DISCUSSION THREAD

**Discussion by billjow** [iNavFlight/inav#7822] [Q&A] [answered]
Hi, I need some help. I have flashed 4.1RC1 to my MambaF722I2C, set up my quad, enabled outputs, set arm mode etc., but it seems to wipe my ESCs. I cannot get the motors to spin on the motors tab and when I come out of iNav configurator and go to the BL Heli configurator, when I read the set up, there is literally nothing there. I have to reflash with Betaflight to get the ESCs and motors to work. BL Heli and Betaflight seem to work fine, so it is not a hardware issue (I have just flown on BF4.3RC3). Is there anything I can try? Is it a "feature" of an unstable release? How do I report it if it is? Thanks, Bill

**Comment by billjow** [comment]
Sorted - please ignore...

**Reply by MrD-RC** [reply]
It may be helpful to say what the issue was. Just in case someone else has a similar problem, or are doing something wrong.

**Comment by billjow** [comment, marked as answer]
Accepted the default Quad 3"-7" settings, enabled outputs, forgot to actually Load and Apply the wizard....Doh! All 4 motors then appeared and was able to test and save. 

Actually, I think that there is a logical/UX design issue here. There is duplication on enabling outputs as there are sliders on the Outputs tab and the Configuration tab. It is also a little bit illogical to accept the defaults on flashing and then have to effectively do the same on the Mixer tab. Perhaps if we reduced the pressure on the devs and gave them time and space, there could be a clean up of the UX.

**Reply by MrD-RC** [reply]
Thanks
